docs/public-networks/how-to/upgrade-node.md

```diff
@@ -6,28 +6,171 @@ tags:
   - public networks
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 # Upgrade your Besu node
 
 When upgrading your Besu node, we recommend:
 
-- Using an orchestration method (for example, Ansible or Chef) to keep all nodes in sync with your desired configuration.
+- Preserving your node's data and configuration
+<!--- - Using an orchestration method  to keep all nodes in sync with your desired configuration. --->  
 - Storing your configuration under version control.
 
-## Ansible
 
-You can use the [Ansible role on Galaxy](https://galaxy.ansible.com/consensys/hyperledger_besu) directly or customize it to suit your needs.
+<Tabs>
+  <TabItem value="binary" label="Linux" default>
+
+### Upgrading on Linux
+
+1. Stop your Besu node:
+
+```bash
+sudo systemctl stop besu
+```
+
+2. Back up your data directory and configuration:
+
+```bash
+sudo cp -r /var/lib/besu/data /var/lib/besu/data.backup
+sudo cp /etc/besu/config.toml /etc/besu/config.toml.backup
+```
+
+3. Download and install the new version:
+
+```bash
+wget https://hyperledger.jfrog.io/artifactory/besu-binaries/besu/[VERSION]/besu-[VERSION].tar.gz
+tar -xvf besu-[VERSION].tar.gz
+sudo cp -r besu-[VERSION]/* /usr/local/bin/
+```
+
+4. Restart your node:
+
+```bash
+sudo systemctl start besu
+```
+
+ :::note
+ Keep your data directory (`/var/lib/besu/data`) intact during upgrades to maintain chain sync status.
+ :::
+ 
+  </TabItem>
+  <TabItem value="docker" label="Docker">
+
+### Upgrading with Docker
+
+1. Update your Docker image:
+
+```bash
+docker pull hyperledger/besu:latest
+```
+
+2. Stop current container:
+
+```bash
+docker stop besu-node
+```
+
+3. Start new container with updated image:
+
+```bash
+docker run -d \
+  --name besu-node \
+  -v besu-data:/opt/besu/data \
+  -v besu-config:/etc/besu \
+  hyperledger/besu:latest
+```
+
+Example `docker-compose.yml`:
+
+```yaml
+version: '3.8'
+services:
+  besu:
+    image: hyperledger/besu:latest
+    volumes:
+      - besu-data:/opt/besu/data
+      - besu-config:/etc/besu
+    ports:
+      - "8545:8545"
+      - "30303:30303"
+volumes:
+  besu-data:
+  besu-config:
+```
 
-Upgrade the Besu version on nodes by running the play with the new version. For more information, For more information, select **Read Me** on the [Ansible Galaxy Besu page](https://galaxy.ansible.com/consensys/hyperledger_besu).
+  </TabItem>
+  <TabItem value="kubernetes" label="Kubernetes">
 
-The playbook:
+### Upgrading in Kubernetes
```

Comment:
I would like someone more familiar with Kubernetes like @joshuafernandes to review this and maybe the docker section please.